src/com/shapeshifters/thecrash/controller/TheCrashApp.java

```diff
@@ -69,6 +68,10 @@ public void execute() {
                         case "quit":
                             setGameOver(true);
                             break;
+                        case "i":
+                            viewInfo();
+                            promptEnterKey();
```

Comment:
I would suggest moving the promptEnterKey(); to the viewInfo() method

Reply:
On it

Reply:
Complete, ty